server/src/main/java/org/apache/iotdb/db/qp/physical/sys/OperateFilePlan.java

```diff
@@ -54,6 +54,11 @@ public OperateFilePlan(File file, File targetDir, OperatorType operatorType) {
     return null;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    return null;
```

Comment:
And there.

Reply:
I'll change it. Thanks for your suggestion.